lib/components/buttons/inverted_button.js

```diff
@@ -2,23 +2,26 @@
 'use strict';
 
 import React from 'react-native';
-const { StyleSheet, Text, TouchableHighlight, Dimensions } = React;
+const { StyleSheet, TouchableHighlight, View, Dimensions } = React;
 
+import Headline from '../text/headline';
 import colors from '../../../data/colors';
 
 const sideMargin = Dimensions.get('window').width > 700 ? 175 : 0;
 
 export default class InvertedButton extends React.Component {
   render() {
     const styling = {
-      underlayColor: (this.props.selected ? colors['purple-regular'] : 'black'),
-      style: [styles.button, { backgroundColor: (this.props.selected ? 'black' : colors['purple-regular']) }],
+      underlayColor: (this.props.selected ? 'black' : colors['purple-regular']),
```

Comment:
so the selected state now goes to black instead of purple, but not yet greyed out (as is stated in [this comment](https://github.com/artsy/eigen/issues/1132#issuecomment-220580717))